README.md

```diff
@@ -52,33 +49,37 @@ kind create cluster
 ```bash
 ./hack/quick_install.sh
 ```
-3) Wait all pods to be ready then launch KFServing `InferenceService`.(wait 1 min for everything to be ready and 40s to
+### Test KFServing Installation 
+
+1) To check if KFServing Controller is installed correctly, please run the following command
+```shell
+kubectl get po -n kfserving-system
+NAME                             READY   STATUS    RESTARTS   AGE
+kfserving-controller-manager-0   2/2     Running   2          13m
+```
+
+Please refer to our [troubleshooting section](docs/DEVELOPER_GUIDE.md#troubleshooting) for recommendations and tips for issues with installation.
+
+2) Wait all pods to be ready then launch KFServing `InferenceService`.(wait 1 min for everything to be ready and 40s to
 launch the `InferenceService`)
 ```bash
 kubectl apply -f docs/samples/sklearn/sklearn.yaml
 ```
-4) Check KFServing `InferenceService` status.
+3) Check KFServing `InferenceService` status.
 ```bash
 kubectl get inferenceservices sklearn-iris
 NAME           URL                                                              READY   DEFAULT TRAFFIC   CANARY TRAFFIC   AGE
 sklearn-iris   http://sklearn-iris.default.example.com/v1/models/sklearn-iris   True    100                                109s
 ```
-5) Curl the `InferenceService`
+4) Curl the `InferenceService`
 ```bash
 kubectl port-forward --namespace istio-system $(kubectl get pod --namespace istio-system --selector="app=istio-ingressgateway" --output jsonpath='{.items[0].metadata.name}') 8080:80
 curl -v -H "Host: sklearn-iris.default.example.com" http://localhost:8080/v1/models/sklearn-iris:predict -d @./docs/samples/sklearn/iris-input.json
 ```
+### KFServing Examples 
+To ensure everything is working, run other [KFServing examples](./docs/samples/README.md)
```

Comment:
Unnecessary verbiage. I'd prefer the link just as before.

Reply:
done